src/types/encodeData/JSONURL.ts

```diff
@@ -9,15 +9,19 @@ interface URLData {
   url: string;
 }
 
+export interface Verification {
```

Comment:
Shouldn't we rename this to something a bit more specific? Like

```suggestion
export interface VerificationOptions {
```

or

```suggestion
export interface VerificationInput {
```

Reply:
Well it's the verification structure, so I think it's clearest if it doesn't have an appendage Options or Info or anything, but that's completely subjective.

Reply:
Ok let's keep it like this then